internal/controller/postgrescluster/pgbackrest_test.go

```diff
@@ -2635,16 +2634,9 @@ volumes:
 	})
 
 	t.Run("PriorityClassName", func(t *testing.T) {
-		cluster := &v1beta1.PostgresCluster{
```

Comment:
:pick: :pick: Part of me wants these refactors as a separate commit, but I should probably stifle that impulse (no change actually requested :stuck_out_tongue_closed_eyes:)

Reply:
👍🏻 Understood. They felt small enough and related enough, but I can keep things like this separate next time. It's easier to squash during review than the reverse.